### PR TITLE
Fixed build on Ubuntu Bionic

### PR DESCRIPTION
**Pull request by ahcorde**
Related with this issue https://github.com/ros2/console_bridge_vendor/issues/23

Signed-off-by: ahcorde <ahcorde@gmail.com>

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -111,7 +111,11 @@ if(BUILD_TESTING)
   if(TARGET benchmark_log)
     target_include_directories(benchmark_log PRIVATE ${console_bridge_INCLUDE_DIRS})
     target_link_libraries(benchmark_log ${console_bridge_LIBRARIES})
-    target_link_directories(benchmark_log PRIVATE ${console_bridge_LIBRARY_DIRS})
+    if(${CMAKE_VERSION} VERSION_LESS "3.13")
+      link_directories(${console_bridge_LIBRARY_DIRS})
+    else()
+      target_link_directories(benchmark_log PRIVATE ${console_bridge_LIBRARY_DIRS})
+    endif()
```
Comment (clalancette) [inline CMakeLists.txt:118]: I'm just curious; is this needed at all?  When I ran it locally commenting out `target_link_directories`, it seemed to build and run just fine for me.

Comment (ahcorde) [reply CMakeLists.txt:118]: This is need it when you are trying to link against `console_bridge` but the install hasn't happend yet. For example: when the workspace is clean or you are compiling for the first time this package

Comment (clalancette) [reply CMakeLists.txt:118]: All right, thanks for the explanation.

**Review by clalancette — APPROVED** [review]

**Comment by ahcorde** [comment]
Building up-to `console_bridge_vendor` and testing `console_bridge_vendor`:

* Linux [Build Status](http://ci.ros2.org/job/ci_linux/12658/)
* Linux-aarch64 [Build Status](http://ci.ros2.org/job/ci_linux-aarch64/7621/)
* macOS [Build Status](http://ci.ros2.org/job/ci_osx/10371/)
* Windows [Build Status](http://ci.ros2.org/job/ci_windows/12593/)

**Pull request merged**